api-catalog-ui/frontend/src/components/ServiceTab/ServiceTab.css

```diff
@@ -86,6 +94,7 @@ p.MuiTypography-root.version-text.MuiTypography-body1 {
     text-align: center;
     /* Data Small/Regular */
     font-size: 12px;
+    text-transform: none;
     line-height: 14px;
     letter-spacing: 0.24px;
     order-radius: 3px;
```

Comment:
it's out of this PR, but I get a warning in my editor about this being an unknown property.